src/components/pages/rules/flyouts/ruleEditor/ruleEditor.container.js

```diff
@@ -13,8 +13,8 @@ const mapStateToProps = state => ({
 
 // Wrap the dispatch method
 const mapDispatchToProps = dispatch => ({
-  insertRule: rule => dispatch(rulesRedux.actions.insertRule(rule)),
-  updateRule: rule => dispatch(rulesRedux.actions.updateRule(rule))
+  insertRules: rules => dispatch(rulesRedux.actions.insertRules(rules)),
+  mergeRules: rules => dispatch(rulesRedux.actions.mergeRules(rules))
```

Comment:
I couldn't keep that naming as it would be a name collision with the one that updates the full list from a server call.  This one updates whatever is already there... and we don't want to overwrite the entire list nor do we want to update last updated date as that's for the data from the server.